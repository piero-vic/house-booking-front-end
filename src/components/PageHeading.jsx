import PropTypes from 'prop-types';

const PageHeading = ({ title, subtitle }) => (
  <div className="mb-5">
    <h2 className="text-center text-black font-black text-3xl">{title}</h2>
    {subtitle !== ''
      && <p className="text-center text-gray-500 font-semibold text-1xl">{subtitle}</p>}
  </div>
);

PageHeading.propTypes = {
  title: PropTypes.string.isRequired,
  subtitle: PropTypes.string,
};

PageHeading.defaultProps = {
  subtitle: '',
};

export default PageHeading;
