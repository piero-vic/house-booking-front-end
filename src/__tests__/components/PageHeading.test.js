import renderer from 'react-test-renderer';
import PageHeading from '../../components/PageHeading';

it('renders correctly', () => {
  const tree = renderer
    .create(<PageHeading title="titleTest" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
