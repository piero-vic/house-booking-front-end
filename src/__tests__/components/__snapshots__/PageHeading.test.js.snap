// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mb-5"
>
  <h2
    className="text-center text-black font-black text-3xl"
  >
    titleTest
  </h2>
</div>
`;
